AGPs/AGP-CRDAO.md

```diff
@@ -0,0 +1,59 @@
+---
+AGP: N/A
+Title:  Community Rewards DAO
+Author: Aaron Foster (`@pythonpete32`) & Burrrata (`@burrrata`)
+Status: Stage IV 
+Track: Finance
+Created: 2019-10-01
+---
+
+# AGP X: Community Rewards DAO
+
+## Address of the transfer recipient
+
+`0xaC3Dbb1302a2319AD9A4359247597f585d75e52C`
+
+## Amount of the transfer
+
+12,000 ANT
+
+## Number and frequency of transfers if recurring (enter “1” if only one payment will be made)
+
+1
+
+## Purpose of the transfer
+
+The CRDAO will reward contributions to the Aragon ecosystem with small amounts of ANT. Its purpose is to reward people for contributions they do and encourage them to do more voluntarily without necessarily an expectation of specific remuneration. it differs from CFDAO in that recipients do not apply for funds, CRDAO rewards are recognition for service to the Aragon Community.
+
+CRDAO will have a fixed weekly budget of 1000 ANT. 1000 ANT will be distributed via the CRDAO as long as sufficient ANT remains in the CRDAO vault. Once Autark's Dot Voting and Allocations apps are published to mainnet, the CRDAO weekly ANT budget will be divided among nominated Aragon community members using Dot voting and Allocations. The reward policy will be based on the [1Hive model](https://1hive.org/contribute/allocations-payroll), however a key difference being that any member of the Aragon forum will be able nominate anyone else, but it is up to the DAO members to actually vote on those nominations. 
+
+Details on the proposed structure, settings, and usage of the CRDAO can be found in the CRDAO GitHub organization:
+- [CRDAO GitHub Org](https://github.com/crdao)
+- [CRDAO deployment guide](https://github.com/crdao/dao/blob/master/README.md)
+- [CRDAO user guide](https://github.com/crdao/dao/blob/master/user-guide.md). 
+- [CRDAO demo live on Rinkeby](https://rinkeby.autark.xyz/#/0xC0d7F939E53de5FE60980cF0357d92230EFB84F3/)
+
+The Aragon community can discuss the CRDAO and submit nominations via the following channels:
+- [#crdao channel on Aragon chat](https://aragon.chat/channel/crdao)
+- [CRDAO thread on the Aragon Forum](https://forum.aragon.org/t/community-rewards-dao/1237)
+
+## Recipient information
+
+Organization (if any)
+Name: Community Rewards DAO
+Website: https://github.com/crdao
+
+Funds will be transferred to an Ethereum address controlled by burrrata. Once Autark's Dot Voting and Allocations apps are published to mainnet the CRDAO will be built and funds will be transferred to the DAO's vault. Burrrata and Aaron will be responsible for creating and managing the DAO. Community members will be asked to inspect and audit the DAO before funds are transferred to ensure permissions are set correctly according to the CRDAO-Outline.
```

Comment:
While I fully expect these apps to go on mainnet soon, perhaps some date could be specified by which if the apps are not deployed on mainnet then the funds will be returned to the Aragon community multisig.

Reply:
Are you implying that there is a non-zero chance that Autark will never ship Open Enterprise to mainnet?

Also, what exactly is that Aragon Community multisig? Is that the AA multisig, or the CFDAO?

Reply:
I updated the proposal with a clause in case Open Enterprise Dot Voting and Allocations apps do not get shipped to mainnet. In this case ANT would be returned to the Aragon Association.

Reply:
> Are you implying that there is a non-zero chance that Autark will never ship Open Enterprise to mainnet?

I'm saying it's an edge case that should be accounted for!

> Also, what exactly is that Aragon Community multisig? Is that the AA multisig, or the CFDAO?

Sorry, meant the AA multisig https://wiki.aragon.org/association/multisigs/association/